### PR TITLE
✨ allow specifying a link for your badge

### PR DESCRIPTION
**Pull request by acidjazz**
Allows you to link your badge to a specified URL - helpful for linking to your test actions

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/AlexBeauchemin/gobadge
+module github.com/acidjazz/gobadge
```
Comment (AlexBeauchemin) [inline go.mod]: Revert this so it can be merged

File: gobadge_test.go [hunk 2]
```diff
@@ -92,11 +92,11 @@ func TestUpdateReadme(t *testing.T) {
...
 
 	createFile(target, "## A Title\nA description\n")
-	err = updateReadme(target, "50.01%", "Coverage", "green")
+	err = updateReadme(target, "50.01%", "Coverage", "green", "")
 	validateFileContent(t, target, "## A Title\n![Coverage](https://img.shields.io/badge/Coverage-50.01%25-green)\nA description\n")
 	assert.Equal(t, nil, err)
 	deleteFile(target)
 
-	err = updateReadme("unknown.md", "50.01%", "Coverage", "green")
+	err = updateReadme("unknown.md", "50.01%", "Coverage", "green", "")
```
Comment (AlexBeauchemin) [inline gobadge_test.go:106]: Add a new test making sure the link is there when url is provided?

**Review by AlexBeauchemin — CHANGES_REQUESTED** [review]
Thanks for your contribution
Looks good to me, just need to revert the module name change and add a test for the url and should be good to merge

**Comment by acidjazz** [comment]
@AlexBeauchemin i think.. i added the test correctly?

**Review by AlexBeauchemin — APPROVED** [review]

**Pull request merged**